pkgs/tools/misc/calamares/default.nix

```diff
@@ -1,24 +1,26 @@
 { lib, fetchurl, boost, cmake, extra-cmake-modules, kparts, kpmcore
-, kservice, libatasmart, libxcb, libyamlcpp, parted, polkit-qt, python, qtbase
-, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
-, ckbcomp, xkeyboard_config, mkDerivation
+, kservice, libatasmart, libxcb, libyamlcpp, libpwquality, parted, polkit-qt, python
+, qtbase, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
+, ckbcomp, xkeyboard_config, mkDerivation, nixos-extensions ? false
 }:
 
 mkDerivation rec {
   pname = "calamares";
-  version = "3.2.44.3";
+  version = "3.2.49.1";
```

Comment:
It seem your image does still have the old version.

![Screenshot from 2022-03-01 22-14-20](https://user-images.githubusercontent.com/91113/156250082-9789135b-616a-4fd8-8da1-b8e934d2a101.png)

Please update after you have incorporated all the suggestions.

Reply:
The image has been updated to version 3.2.52